.github/actions/install_requirements/action.yml

```diff
@@ -0,0 +1,34 @@
+name: Install requirements
+description: Install a version of python then call pip install and report what was installed
+inputs:
+  python-version:
+    description: Python version to install, default is from Dockerfile
+    default: "dev"
+  pip-install:
+    description: Parameters to pass to pip install
+    default: "$([ -f dev-requirements.txt ] && echo '-c dev-requirements.txt') -e .[dev]"
```

Comment:
It's now called `dev-requirements.txt` as dependabot only looks at `requirements*.txt` files. Also I think we intend to move its contents into a `dev` extra so this file should be used as a lockfile in case we want one...

Reply:
Its contents have already been moved to the dev extra

Reply:
Good enough. I am not sure if we need those lines, but they are also conditional, so we can leave it as is.